modules/smart-agent_cassandra-nodetool/conf/00-status.yaml

```diff
@@ -0,0 +1,13 @@
+## Example
+module: cassandra nodetool
+name: "Node is not in live status"
+transformation: ".min(over='10m')"
+aggregation: false
+signals:
+  signal:
+    metric: "cassandra.status"
+rules:
+  critical:
```

Comment:
I would keep 2 rules different to have 2 different severities for different "abnormal" statuses.

You was closer with before last change:
- you just had to customize the description like `is dead` && `is unknown` keeping `<` and rules dependency as it was.

but you can keep the new more "strict" logic:
- removing dependency between them just, a different severity like `critical` for dead and `minor` for unknown
- then you can use `== 0` && `== -1` for dead and unknown
- as for the first solution : update the description now it is possible